options.html

```diff
@@ -51,13 +66,15 @@
         Temporarily disable RECAP uploading even when logged in to PACER
       </div>
     </label>
+  </fieldset>
 
-    <p id="receipts_disabled" class="alert hidden"><i class="fa fa-warning gold"></i>&nbsp;Receipts are disabled in your PACER profile. While receipts are disabled, we are unable to upload PDFs that you purchase. To change this setting, please go to the billing preferences in your PACER account.
-    </p>
+  <p id="receipts_disabled" class="alert hidden"><i class="fa fa-warning gold"></i>&nbsp;Receipts are disabled in your PACER profile. While receipts are disabled, we are unable to upload PDFs that you purchase. To change this setting, please go to the billing preferences in your PACER account.
+  </p>
   </div>
   <div id="donate-plea" class="text-center">
-    <a href="https://free.law/donate/?referrer=recap-extension">
-      <img src="assets/images/donate-button.png">
+    <a href="https://free.law/donate/?referrer=recap-extension" style="text-decoration: none">
+      <img class="pad20" src="assets/images/donate-button.png">
+      <img class="pad20" width="150" src="assets/images/webstore-exports/440x280.png">
```

Comment:
I haven't looked at how this loads, but do we need a properly scaled down version of the RECAP logo? It's in the assets folder as an SVG that you can convert to whatever size. I'm happy to do so. I guess we need one that's 85px tall?